tests/transforms/test_hamiltonian_expand.py

```diff
@@ -407,10 +407,10 @@ def test_constant_offset_no_grouping(self):
 
 with AnnotatedQueue() as s_tape1:
     qml.PauliX(0)
-    S1 = qml.s_prod(1.5, qml.prod(qml.PauliZ(0), qml.PauliZ(1)))
-    qml.expval(S1)
+    S1 = qml.s_prod(1.5, qml.sum(qml.prod(qml.PauliZ(0), qml.PauliZ(1)), qml.Identity()))
     qml.expval(S1)
     qml.state()
+    qml.expval(S1)
```

Comment:
expval(S1) twice in tape, is that intentional?

Reply:
I believe yes, probably to test that the same observables are reused for multiple measurements.